decode/render.ts

```diff
@@ -113,12 +115,12 @@ function css(style: CSSStyleDeclaration, field: string): number {
 
 export function markup(type: Event, data: DomData[], iframe: HTMLIFrameElement): void {
     let doc = iframe.contentDocument;
-    if (type === Event.Discover) { reset(); }
     for (let node of data) {
         let parent = element(node.parent);
         let next = element(node.next);
         switch (node.tag) {
             case "*D":
```

Comment:
What do these cases mean and what are the tags from? Could you make these more meaningfully named constants?

Reply:
This is playback code. Earlier discover event was sent only once and whenever we received it, it would mean we are dealing a new impression (mostly for extension). With introduction of shadow dom it no longer is true so I removed that condition.